docs/services/topology-contacts-data.md

```diff
@@ -0,0 +1,117 @@
+The YAML file contains sensitive information and is only visible to people with access to that repo.
+
+
+### Getting access to the contact repo
+
+The contacts repo is hosted on BitBucket.
+You will need an Atlassian account for access to BitBucket.
+The account you use for OSG JIRA should work.
+Once you have an account, request access from Brian Lin or Mat Selmeci.
+You should then be able to go to <https://bitbucket.org/opensciencegrid/contact/>.
+
+
+### Using the contact repo
+
+BitBucket is similar to GitHub except you don't make a fork of the contact repo,
+you just clone it to your local machine.
+This means that any pushes go directly to the main repo instead of your own fork.
+
+!!!note
+    Don't push to master.
+    For any changes, always create your own branch, push your changes to that branch, then make a pull request.
+    Have someone else review and merge your pull request.
+
+All contact data is stored in `contacts.yaml`.
+The contact info is keyed by a 40-character hexadecimal ID which was generated from their email address when they were first added.
+An example entry is:
+```yaml
+25357f62c7ab2ae11ddda1efd272bb5435dbfacb:
+# ^ this is their ID
+  FullName: Example A. User
+  Profile: This is an example user.
+  GitHub: ExampleUser
+  # ContactInformation data requires authorization to view
+  ContactInformation:
+    DNs:
+    - ...
+    IM: ...
+    PrimaryEmail: user@example.net
+    PrimaryPhone: ...
+```
+
+When making changes to the contact data, first see if a contact is already in the YAML file.
+Search the YAML file for their name.
+Be sure to try variations of their name if you don't find them --
+someone may be listed as "Dave" or "David", or have a middle name or middle initial.
+
+Follow the instructions below for adding or updating a contact, as appropriate.
+
+
+#### Adding a new contact
+
+If the contact does not exist in the `contacts.yaml` file, fill out the values in `template-contacts.yaml` and add it to `contacts.yaml`.
+To get the hash used as the ID, run `email-hash` on their email address.
+For example:
+
+```command
+$ bin/email-hash user@example.net
+25357f62c7ab2ae11ddda1efd272bb5435dbfacb
+```
+Then your new entry will look like
+```yaml
+25357f62c7ab2ae11ddda1efd272bb5435dbfacb:
+    FullName: Example A. User
+    ....
+```
+
+The `FullName` field and the `PrimaryEmail` field in the `ContactInformation` section are required.
+`PrimaryEmail` field in the `ContactInformation` section should match the hash that you used for the ID.
+
+
+#### Editing a contact
+
+Once you have found a contact in the YAML file, edit the attributes by hand.
+If you want to add information that is not present for that contact, look at `template-contacts.yaml` to find out what the attributes are called.
+
+!!!note
+    The ID of the contact never changes, even if the user's `PrimaryEmail` changes.
+
+!!!note
+    If you change the contact's `FullName`, you should make the same change to every place that the contact
```

Comment:
~you should~

Reply:
A name/ID mismatch will soon be an error so maybe this should be "you **must**"?

Reply:
I think just dropping "you should" has the same effect of indicating a requirement. Either way works for me, though.